src/fluree/db/serde/avro.clj

```diff
@@ -153,7 +153,7 @@
     {:type      :record
      :name      "FdbRoot"
      :namespace "fluree"
-     :fields    [{:name "dbid", :type :string}
+     :fields    [{:name "ledger-id", :type :string}
```

Comment:
we're probably going to have to figure out another migration path for these. it might "just work" with the current 2.0 migration, but i'm guessing that the reindex process could choke when trying to load the old database roots with this schema. this can definitely wait for another pr, but we should keep this in mind.

Reply:
Oh, hmm... My intention is definitely not to break things that should keep working. I'll look into it. Thanks for catching!

Reply:
I think new ledgers will work fine, but there is a chance that 1.0 ledgers will need another step to migrate to 2.0 to continue working, or it might just work automatically.

Reply:
Also, we already have to run a reindex migration to convert 1.0 ledgers to 2.0 anyway, so anything else we have to do for this can just be added to that process.

Reply:
Tested this locally and it seemed to work OK.

I created a ledger on 1.0.1, transacted some data into it, ran my PR code on it, reindexed, got no errors on that, and then queried out my data successfully.